client/src/test/kotlin/no/nav/su/se/bakover/client/oppgave/OppgaveHttpClientTest.kt

```diff
@@ -815,7 +816,91 @@ internal class OppgaveHttpClientTest : WiremockBase {
                 aktørId = AktørId(aktørId),
                 tilordnetRessurs = null,
             ),
-        ) shouldBeLeft KunneIkkeOppretteOppgave
+        ) shouldBeLeft OppgaveFeil.KunneIkkeOppretteOppgave
+    }
+
+    @Test
+    fun `oppdaterer eksiterende oppgave med ny beskrivelse`() {
```

Comment:
Litt usikker på om denne egentlig gjør noe fornuftig med tanke på at vi bare kan sjekke at vi får en Unit tilbake?

Reply:
Kan nok vurdere å bare droppe testen ja, siden den bare kaller videre på noe eksternt som dere mocker. Evt. kan man kanskje sjekke noe på `OppdatertOppgaveResponse`? Ellers går det nok å gjøre noe triks for å sjekke at beskrivelse og status og sånt på requesten blir satt slik du forventer (Mockito har f.eks. et begrep om ArgumentCaptors, hvor man kan sjekke hvordan et argument til en mocket tjeneste ser ut).